### PR TITLE
make closure test errors fatal; fix errors on master

### PR DESCRIPTION
**Pull request by brendankenny**

### CODE REVIEW

File: lighthouse-core/report/v2/renderer/category-renderer.js [hunk 1]
```diff
@@ -233,21 +233,23 @@ class CategoryRenderer {
     element.id = category.id;
     element.appendChild(this._renderCategoryScore(category));
 
-    const auditsGroupedByGroup = category.audits.reduce((indexed, audit) => {
+    const auditsGroupedByGroup = /** @type {!Object<string,
+        {passed: !Array<!ReportRenderer.AuditJSON>,
+        failed: !Array<!ReportRenderer.AuditJSON>}>} */ ({});
```
Comment (brendankenny) [inline lighthouse-core/report/v2/renderer/category-renderer.js]: open to less awkward type cast here, but not sure how. One option is to split `auditsGroupedByGroup` into two `!Object<string, !Array<!ReportRenderer.AuditJSON>>`, one for passes and one for failed, but wanted to minimize change to @patrickhulce's code

**Comment by brendankenny** [comment]
hold off for a second, intentionally reintroducing error to check that it catches it :)

**Comment by brendankenny** [comment]
confirmed it fails the travis run now, ready for review :)

File: lighthouse-core/report/v2/renderer/category-renderer.js (hunk 1 above)
Comment (patrickhulce) [inline lighthouse-core/report/v2/renderer/category-renderer.js]: eh I'm cool with splitting into passed and failed arrays instead if it makes it cleaner here

**Comment by brendankenny** [comment]
> I still get 46 errors complaining

I think you might need to `yarn`

File: lighthouse-core/report/v2/renderer/category-renderer.js (hunk 1 above)
Comment (paulirish) [inline lighthouse-core/report/v2/renderer/category-renderer.js]: this typecast lgtm. its quite clear, i'd say.

File: lighthouse-core/closure/closure-type-checking.js [hunk 2]
```diff
@@ -51,7 +51,7 @@ gulp.task('compile-report', () => {
     // new_type_inf: true,
     language_in: 'ECMASCRIPT6_STRICT',
     language_out: 'ECMASCRIPT5_STRICT',
-    warning_level: process.env.CI ? 'QUIET' : 'VERBOSE',
+    warning_level: 'VERBOSE',
```
Comment (paulirish) [inline lighthouse-core/closure/closure-type-checking.js]: :) good call

**Pull request merged**